pyproject.toml

```diff
@@ -224,6 +212,7 @@ pytest-cov = "^5.0.0"
 pytest-env = "^1.1.1"
 fuzzywuzzy = "^0.18.0"
 pytest-clarity = "^1.0.1"
+torch = "^2.4.1"
```

Comment:
Kind of clunky moving this to `test` deps but that's the only time we require it.